### PR TITLE
jpe/debuggingtutorial

### PR DESCRIPTION
**Pull request by jpe442**
## Urgency
- [ ] Blocker 
- [ x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by jpe442** [comment]
I think it is looking good enough to put up. Some of the links don't work on the local build due to redirects not being enabled locally, but they should work when promoted.

**Comment by jpe442** [comment]
okay, now I think it is actually good enough to go up 👍 but I would love a quick review from Jorg if possible.

File: pages/1.11/deploying-services/fault-domain-awareness/index.md [hunk 1]
```diff
@@ -59,9 +59,22 @@ You can deploy your instances in a specific region region based on the available
 
 1. Create a fault domain detect script to run on each node to detect the node's fault domain (Enterprise only). During installation, the output of this script is passed to Mesos.
 
-   We recommend the format for the script output be `fault_domain: region: name: <region>, zone: name: <zone>` We provide [fault domain detect scripts for AWS and Azure nodes](https://github.com/dcos/dcos/tree/master/gen/fault-domain-detect). For a cluster that has aws nodes and azure nodes you would combine the two into one script. You can use these as a model for creating a fault domain detect script for an on premises cluster.
+We recommend the format for the script output be:
 
-   <table class="table" bgcolor="#FAFAFA"> <tr> <td style="border-left: thin solid; border-top: thin solid; border-bottom: thin solid;border-right: thin solid;"><b>Important:</b> This script will not work if you use proxies in your environment. If you use a proxy, modifications will be required.</td> </tr> </table>
+```json
```
Comment (joerg84) [inline pages/1.11/deploying-services/fault-domain-awareness/index.md]: Should this be part of this PR?

Comment (jpe442) [reply pages/1.11/deploying-services/fault-domain-awareness/index.md]: I have confirmed that these changes are already live. They have been having odd Jenkins caching issues between staging and master - not sure if this is related.

Comment (jpe442) [reply pages/1.11/deploying-services/fault-domain-awareness/index.md]: https://docs.mesosphere.com/1.11/deploying-services/fault-domain-awareness/#installation-steps

**Pull request closed without merging**